tests-legacy/Integration/PrestaShopBundle/Test/LightWebTestCase.php

```diff
@@ -132,6 +133,13 @@ protected function setUp()
             ->setMethods(['getDefaultCurrencyIsoCode', 'getDefaultCurrency'])
             ->getMock();
 
+        $linkMock = $this
+            ->getMockBuilder(Link::class)
+            ->disableOriginalConstructor()
+            ->getMock();
+
+        $contextMock->link = $linkMock;
```

Comment:
@matks it turns out that legacy tests did not had link mocked - I added it so it will now exist in the context :)